lib/blocks.js

```diff
@@ -787,20 +782,32 @@ var blockStarts = [
             return 0;
         }
 
+        const nextColumn = measureNonspaceColumn(parser.nextLine);
```

Comment:
Ensuring that the second line of the table is near the same indentation level as the first line was previously part of `reTableDelimiter`, but it needs to be done manually now